float32 p_z
float32 d_z
float32 p_rot
float32 d_rot
float32 p_trans
float32 d_trans
float32 i_action
float32 v_damping
bool i_enabled
bool v_enabled